hnn_core/parallel_backends.py

```diff
@@ -11,7 +11,7 @@
 import codecs
 from warnings import warn
 from subprocess import Popen
-import select
+import selectors
```

Comment:
[`selectors`](https://docs.python.org/3/library/selectors.html) is a higher-level module than `select`